akka-http/src/main/scala/akka/http/scaladsl/server/directives/CodingDirectives.scala

```diff
@@ -134,7 +134,7 @@ trait CodingDirectives {
 
   /**
    * Inspects the response entity and adds a `Content-Encoding: gzip` response header if
-   * the entities media-type is precompressed with gzip and no `Content-Encoding` header is present yet.
+   * the entity's media-type is precompressed with gzip and no `Content-Encoding` header is present yet.
```

Comment:
sure? I thought entity's is "entity is" and the other one is possesive? Do I keep getting this wrong still? ;)